src/components/movie-card/movie-card.tsx

```diff
@@ -13,15 +15,75 @@ export const MovieCard: FC<MovieCardProps> = ({
   title,
   year,
   description,
-}) => (
-  <div className={styles.card}>
-    <img src={img} alt={`${title}-poster`} className={styles.img} />
-    <div className={styles.descriptionWrappper}>
-      <div className={styles.nameWrapper}>
-        <div className={styles.title}>{title}</div>
-        <div className={styles.year}>{year}</div>
+}) => {
+  const [showMenu, setShowMenu] = useState<boolean>(false);
+  const [showEditModal, setShowEditModal] = useState<boolean>(false);
+  const [showDeleteModal, setShowDeleteModal] = useState<boolean>(false);
+
+  const closeMenu = () => {
+    setShowMenu(false);
+  };
+
+  const handleEdit = () => {
```

Comment:
Guess `handle` is not too suitable here, you can easily use `openEditModal` and `closeEditModal`, it will simplify code reading